datafusion/expr/src/registry.rs

```diff
@@ -27,9 +27,25 @@ use std::sync::Arc;
 
 /// A registry knows how to build logical expressions out of user-defined function' names
 pub trait FunctionRegistry {
-    /// Set of all available udfs.
+    /// Returns names of all available scalar user defined functions.
     fn udfs(&self) -> HashSet<String>;
 
+    /// Returns names of all available aggregate user defined functions.
+    fn udafs(&self) -> HashSet<String> {
+        // This default implementation is provided temporarily
```

Comment:
👍

Reply:
A bit confused by your comment @alamb not sure how is this functionality related to ud(a|w)?f serialisation. 

If we want to demonstrate ud(a|w)?f serialisation maybe something like https://github.com/milenkovicm/ballista_python/blob/0398011f2b200dd44dd91e3da5de89e863a45c71/src/codec.rs#L197-L220 could be done?